jina/executors/indexers/cache.py

```diff
@@ -10,28 +12,38 @@
 
 
 class BaseCache(BaseKVIndexer):
-    """Base class of the cache inherited :class:`BaseKVIndexer`
+    """Base class of the cache inherited :class:`BaseKVIndexer`.
 
     The difference between a cache and a :class:`BaseKVIndexer` is the ``handler_mutex`` is released in cache,
     this allows one to query-while-indexing.
     """
 
     def __init__(self, *args, **kwargs):
+        """Create a new BaseCache."""
         super().__init__(*args, **kwargs)
 
     def post_init(self):
+        """Call the post-init."""
```

Comment:
I think this does not make much sense right? I think may not be needed

Reply:
Yes, will move the comment here